help/search-for-messages.md

```diff
@@ -41,9 +41,7 @@ Some details to keep in mind:
 - Zulip search ignores very common words like `a`, `the`, and about 100 others.
 - [Emoji](/help/emoji-and-emoticons) in messages (but not [emoji
   reactions](/help/emoji-reactions)) are included in searches, so if you search
-  for `octopus`, your results will include messages with the `:octopus:` emoji (
-  <img src="/static/generated/emoji/images-google-64/1f419.png" alt="octopus"
-  class="emoji-small"/>).
+  for `octopus`, your results will include messages with the `:octopus:` emoji (🐙).
```

Comment:
Should we choose an emoji here that doesn't vary in design so much?

Reply:
Not sure it's worth worrying about? But it's fine to change it as well.

Reply:
Fixed at b5b829c821b1a41606b09858ba1e90cf25079a85